DeepSpeech.ipynb

```diff
@@ -84,14 +84,16 @@
     "import subprocess\n",
     "import numpy as np\n",
     "from math import ceil\n",
+    "from xdg import BaseDirectory as xdg\n",
     "import tensorflow as tf\n",
     "from util.log import merge_logs\n",
     "from util.gpu import get_available_gpus\n",
     "from util.text import sparse_tensor_value_to_texts, wers\n",
     "from tensorflow.python.ops import ctc_ops\n",
+    "from tensorflow.contrib.session_bundle import exporter\n",
     "\n",
     "ds_importer = os.environ.get('ds_importer', 'ted_lium')\n",
-    "ds_dataset_path = os.environ.get('ds_dataset_path', './data/ted')\n",
+    "ds_dataset_path = os.environ.get('ds_dataset_path', os.path.join('./data', ds_importer))\n",
```

Comment:
It's unrelated, was just annoyed at how many times I got caught out changing ds_importer and forgetting to change ds_dataset_path. I've also got the changeset that makes us use XDG_DATA_HOME instead of /tmp in here too, shall I split them out into a separate PR?